src/app/book/booklist/booklist.component.html

```diff
@@ -0,0 +1,25 @@
+<div>
+  <table class="table table-bordered">
+  <thead>
+    <tr>
+      <th>Book Image</th>
+      <th>Book Title</th>
+      <th>Book Price</th>
+      <th>Book Rating</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr *ngFor = "let book of books" >
+      <!-- <td>
```

Comment:
remove commented code

Reply:
done. yes I have created branch before merging. and I just add router link to get bookid with route which is in booklist component to pass it in bookdetails component. ('book/{{ book.bookId }}') thats why changes made in booklist.html.